### PR TITLE
Use Voila logo for the spinner

### PR DESCRIPTION
**Pull request by martinRenou**
spinner

### CODE REVIEW

**Comment by jtpio** [comment]
That looks nice.

Maybe we want to minify / compress the svg file too ?

**Comment by martinRenou** [comment]
We can try, but it looks pretty small already

**Comment by SylvainCorlay** [comment]
This is really cool!

The improvement of the spinner SVG should probably be in a different PR. (Removing the green circle under the yellow will remove the rendering artifact)

**Comment by SylvainCorlay** [comment]
A way to prevent the loading delay may be to inline the SVG in the HTML.

**Comment by SylvainCorlay** [comment]
One simple change to make that will reduce the size would be to remove all the attributes prefixed with `inkscape` in the SVG.

**Comment by jtpio** [comment]
Inkscape has an option to save as "Optimized SVG (*.svg)", which should take care of all these.

**Comment by martinRenou** [comment]
I also removed unused defined colors, whitespaces and new lines, saving ~100kb

**Comment by jtpio** [comment]
Does this affect the perceived loading delay?

File: share/jupyter/voila/templates/default/nbconvert_templates/voila.tpl [hunk 1]
```diff
@@ -15,6 +15,23 @@
...
 
+<style>
+  .spinner {
+      -webkit-animation: rotation 2s infinite linear;
+    -moz-animation: rotation 2s infinite linear;
+    -o-animation: rotation 2s infinite linear;
+    animation: rotation 2s infinite linear;
+      transform-origin: 50% 50%;
+      -webkit-transform-origin: 50% 50%;
+      -moz-transform-origin: 50% 50%;
```
Comment (SylvainCorlay) [inline share/jupyter/voila/templates/default/nbconvert_templates/voila.tpl]: I am not sure you still need the `-moz`, `-o` and `-webkit` versions of css properties.

Comment (martinRenou) [reply share/jupyter/voila/templates/default/nbconvert_templates/voila.tpl]: Fixed :) Thanks

**Comment by martinRenou** [comment]
> Does this affect the perceived loading delay?

It does actually, the logo arrived the same time as the text when I tried. But it's only local...

**Comment by SylvainCorlay** [comment]
The way to fix this is to inline the svg in the HTML.

This will also make it possible to set the colors of the parts of the spinner with CSS.

File: share/jupyter/voila/templates/default/nbconvert_templates/voila.tpl [hunk 2]
```diff
@@ -42,6 +54,14 @@ a.anchor-link {
         color: var(--jp-content-font-color1);
         font-family: sans-serif;
     }
+
+    .color1{
```
Comment (SylvainCorlay) [inline share/jupyter/voila/templates/default/nbconvert_templates/voila.tpl]: `voila-spinner-color1`, `voila-spinner-color2`...

**Comment by SylvainCorlay** [comment]
This looks great and I am happy to get this in as it is now.

Leaving it open to give a chance to @maartenbreddels to chime in.

**Comment by martinRenou** [comment]
Some CSS fixes coming

**Comment by jtpio** [comment]
Anything more to add / fix before merging it? (so it can go in the next release)

**Comment by SylvainCorlay** [comment]
We should credit Jasmine for her contribution!

**Comment by maartenbreddels** [comment]
Perfect, especially inlining the svg indeed, that should render instantly.

**Comment by martinRenou** [comment]
@jasminefrh I added your contribution :) Thanks a lot for your help!

**Pull request merged**